timon/plugins/http_server/views.py

```diff
@@ -94,3 +133,32 @@ async def force_probe_run(probename):
         "msg": probe.msg,
     }
     return data_to_return
+
+
+@app.route("/rescheduler/probes/", methods=['POST'])
+async def reschedule_probes():
+    """
+    Reschedule a specific probe to have
+    Params in request body:
+        - probenames
+        - timestamp (optional, default=time.time())
+    """
+    strdata = await request.get_data()
+    data = json.loads(strdata)
+    probenames = data["probenames"]
+    new_schedular = data.get("timestamp", time.time())
```

Comment:
schedular -> scheduler

Reply:
and by the way this is not a new scheduler (the one who organizes the schedule)
it is just a time .
so new_time or new_sched_time or new_scheduled_time